index.html

```diff
@@ -16,16 +16,16 @@ <h2>Filter Starred Ideas</h2>
     </aside>
     <section class="column2">
       <section class='input-boxes'>
-        <label>Title</label>
+        <label class="label">Title</label>
```

Comment:
We definitely need to do some renaming when we refactor, some of these aren't super specific.

Reply:
I agree, need to make them more specific, i.e. label-title instead of label